invenio_search/walkers/elasticsearch.py

```diff
@@ -81,6 +81,11 @@ def visit(self, node, left, right):
             keyword = self.map_keyword_to_fields(
                 left, getattr(right, '__search_mode__', 'a')
             )
+            if keyword == ['authors.full_name']:
+                from inspirehep.modules.authors.utils import author_tokenize
+                name_variations = author_tokenize(node.right.value)
+                return {"bool": {"must": [{"terms": {"authors.name_variations": name_variations}}], "should": [
```

Comment:
inspirehep/invenio-search#17 takes them into account. This pr should be closed but I left it open because the conversation was taking place here.